plugins/DigitalLibrary/resources/qml/SaveProjectFilesPage.qml

```diff
@@ -208,7 +208,14 @@ Item
         anchors.rightMargin: UM.Theme.getSize("thin_margin").height
 
         enabled: UM.Backend.state == UM.Backend.Done
-        currentIndex: UM.Backend.state == UM.Backend.Done ? (Cura.MachineManager.activeMachine.getOutputFileFormats.includes("application/x-makerbot") ? 1 : 0) : 2
+
+        // Pre-select the correct index, depending on the situation (see the model-property below):
+        // - Don't select any post-slice-file-format when the engine isn't done.
+        // - Choose either the S-series or the Makerbot-series of printers' format otherwise, depending on the active printer.
+        // This way, the user can just click 'save' without having to worry about wether or not the format is right.
+        property int isMakerbotFormat: Cura.MachineManager.activeMachine.getOutputFileFormats.includes("application/x-makerbot") || Cura.MachineManager.activeMachine.getOutputFileFormats.includes("application/x-makerbot-sketch")
```

Comment:
Is there a place we could get the makerbot mime types from ? If we add a third one in some months, it is very likely that we will do the very same mistake...

Reply:
I suppose we could make it into a method somewhere and mark that as a pyqtproperty.

I'm not sure how likely it is that we even continue with any sort of `.makerbot` filetype beyond what the Method and Sketch lines have now.

Reply:
Indeed, we can hope very hard that we don't have a 3rd Makerbot file format :sweat_smile: let's don't change this then.